.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+on:
+  push:
```

Comment:
Will this still re-run CI if you push commits to a PR? For example, if tests fail and you push a new commit to fix the test, I think we want CI to run again?

Reply:
Every time we push a commit to this PR, ci will re-run. You can see the little check next to every commit:

<img width="1088" alt="Screen Shot 2021-05-25 at 4 46 34 PM" src="https://user-images.githubusercontent.com/22174780/119566139-cee87c80-bd78-11eb-97c4-efa6643eacba.png">

Is that what you mean? 🤔

Reply:
Nice yea!